+ Patch to remove the EFAULT related sub test as it currently causes
+ enclave abort due to a product issue 169

diff --git a/testcases/kernel/syscalls/getsockname/getsockname01.c b/testcases/kernel/syscalls/getsockname/getsockname01.c
index cce1543fd..c8880b3a7 100644
--- a/testcases/kernel/syscalls/getsockname/getsockname01.c
+++ b/testcases/kernel/syscalls/getsockname/getsockname01.c
@@ -83,21 +83,23 @@ struct test_case_t {		/* test case structure */
 	PF_INET, SOCK_STREAM, 0, (struct sockaddr *)&fsin1,
 		    &sinlen, -1, ENOTSOCK, setup0, cleanup0,
 		    "bad file descriptor"},
-#ifndef UCLINUX
-	    /* Skip since uClinux does not implement memory protection */
-	{
-	PF_INET, SOCK_STREAM, 0, NULL,
-		    &sinlen, -1, EFAULT, setup1, cleanup1,
-		    "invalid socket buffer"}, {
-		/* invalid salen test for aligned input */
-	PF_INET, SOCK_STREAM, 0, (struct sockaddr *)&fsin1,
-		    NULL, -1, EFAULT, setup1, cleanup1,
-		    "invalid aligned salen"}, {
-		/* invalid salen test for unaligned input */
-	PF_INET, SOCK_STREAM, 0, (struct sockaddr *)&fsin1,
-		    (socklen_t *) 1, -1, EFAULT, setup1, cleanup1,
-		    "invalid unaligned salen"},
-#endif
+//TODO: uncomment below 15 lines after fixing github issue 169.
+//url: https://github.com/lsds/sgx-lkl/issues/169
+//#ifndef UCLINUX
+//	    /* Skip since uClinux does not implement memory protection */
+//	{
+//	PF_INET, SOCK_STREAM, 0, NULL,
+//		    &sinlen, -1, EFAULT, setup1, cleanup1,
+//		    "invalid socket buffer"}, {
+//		/* invalid salen test for aligned input */
+//	PF_INET, SOCK_STREAM, 0, (struct sockaddr *)&fsin1,
+//		    NULL, -1, EFAULT, setup1, cleanup1,
+//		    "invalid aligned salen"}, {
+//		/* invalid salen test for unaligned input */
+//	PF_INET, SOCK_STREAM, 0, (struct sockaddr *)&fsin1,
+//		    (socklen_t *) 1, -1, EFAULT, setup1, cleanup1,
+//		    "invalid unaligned salen"},
+//#endif
 };
 
 int TST_TOTAL = sizeof(tdat) / sizeof(tdat[0]);
